docker-compose.ccm.yaml

```diff
@@ -39,6 +39,20 @@ services:
       - traefik.http.middlewares.app.headers.accessControlAllowCredentials=true
       - traefik.http.middlewares.app.headers.frameDeny=true
       - traefik.http.middlewares.app.headers.browserXssFilter=true
```

Comment:
The container labels are equivalent to declaring these in separate YAML files like in `traefik/`
Since the `app` middleware is already declared here, we can reuse it elsewhere as the configuration elements aren't actually scoped to the container but merged into one.